<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Computer</class>
 <widget class="QDialog" name="Computer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Love Letter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/ressources/game_icon.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
background-color: rgb(81, 60, 117);
*/</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>960</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>ENTER YOUR NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>140</y>
     <width>841</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="easyButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string>EASY</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="mediumButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string>MEDIUM</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="hardButton">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:checked{
	background-color: rgb(227,91,120);
	color: rgb(255,229,232);
}</string>
      </property>
      <property name="text">
       <string>HARD</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>791</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;

</string>
      </property>
      <property name="text">
       <string>NAME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255,229,232);
	color: rgb(227,91,120);
}

QPushButton:disabled{
	background-color: rgb(194,165,171);
	color: rgb(79,63,66);
}</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>960</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>SELECT MODE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
